linera-rpc/proto/rpc.proto

```diff
@@ -116,6 +116,15 @@ message CertificatesBatchResponse {
   repeated Certificate certificates = 1;
 }
 
+message CertificatesBatch {
+  repeated CertificateRaw certificates = 1;
+}
+
+message CertificateRaw {
```

Comment:
nit: naming -> `RawCertificate`

It's quite possible that we don't need the other `Certificate` at all. I looked at `GrpcProxyable` and it's not using the specific fields.